pkg/networkmanager/firewall.jsx

```diff
@@ -921,9 +921,9 @@ export class Firewall extends React.Component {
                 </ol>
                 <h1>
                     {_("Firewall")}
-                    <OnOffSwitch state={enabled}
-                                 disabled={!!this.state.pendingTarget}
-                                 onChange={this.onSwitchChanged} />
+                    <Switch id='firewall-onoff-switch' isChecked={enabled}
+                        isDisabled={this.state.pendingTarget}
```

Comment:
Let's please keep the `!!` here. pendingTarget can be `null`, which is not a valid boolean.